errors/runconfig.go

```diff
@@ -0,0 +1,204 @@
+package errors
+
+// This file contains all of the errors that can be generated from the
+// docker/daemon component.
+
+import (
+	"net/http"
+
+	"github.com/docker/distribution/registry/api/errcode"
+)
+
+var (
+	// ErrorCodeInvalidMacAddress is generated when invalid Mac Address is provided.
+	ErrorCodeInvalidMacAddress = errcode.Register(errGroup, errcode.ErrorDescriptor{
+		Value:          "INVALIDMACADDRESS",
```

Comment:
yeah or with underscores, but I see we already have it like that in errors/ so it's out of scope for this PR, was just curious.